lib/std/process.zig

```diff
@@ -515,35 +515,36 @@ pub fn argsAlloc(allocator: *mem.Allocator) ![][]u8 {
     while (it.next(allocator)) |arg_or_err| {
         const arg = try arg_or_err;
         defer allocator.free(arg);
-        try contents.appendSlice(arg);
+        try contents.appendSlice(@ptrCast([*]u8, arg.ptr)[0 .. arg.len + 1]);
```

Comment:
Why is this cast required?

Reply:
Oh it's not. I assumed that trying to slice one past the end of the slice would trigger bounds-checking, so I put the cast there. But just realized it's not needed. Will fix.